File: ruby/measures/test_measure.rb
class RubyTestMeasure < Openstudio::Measure

  def name
    return "Ruby Test Measure"
  end

  def run(runner)
    m = runner.get_current_model()
    obj = Openstudio::ModelObject.new(m)
    obj.setName("ModelObject created in Ruby");

    f = Openstudio::Foo.new()
    bars = f.bars()
    bars.each do |b|
      puts b.name()
    end

    return true;
  end
end

File: python/measures/test_measure.py
import openstudio

class PythonTestMeasure(openstudio.Measure):
    def __init__(self):
        openstudio.Measure.__init__(self)

    def run(self, r: openstudio.Runner):
        m = r.get_current_model()
        obj = openstudio.ModelObject(m)
        obj.setName("ModelObject created in Python")

        f = openstudio.Foo()
        bars = f.bars()
        for b in bars:
          print(b.name())

        return True

    def name(self):
        return "Python Test Measure"
